testsuite/tests/singlecluster/mtls/test_mtls_configuration_with_authpolicy.py

```diff
@@ -0,0 +1,67 @@
+"""
+Tests mTLS configuration and readiness after Kuadrant CR changes (authpolicy)
+"""
+
+import pytest
+
+from openshift_client import selector
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.disruptive]
+
+
+@pytest.fixture(scope="module")
+def rate_limit(cluster, blame, module_label, gateway, route):  # pylint: disable=unused-argument
+    """RateLimitPolicy for testing"""
+    return None
+
+
+def test_peer_authentication_resource_applied_auth_policy(
+    kuadrant, cluster, authorization, wait_for_mtls_status, reset_mtls
+):  # pylint: disable=unused-argument
+    """Verify PeerAuthentication with STRICT mode is created when mTLS is enabled"""
+    kuadrant.refresh().model.spec.mtls = {"enable": True}
+    kuadrant.apply()
+    kuadrant.wait_for_ready()
+    wait_for_mtls_status(kuadrant, expected=True, component="authorino")
+
+    project = cluster.change_project("kuadrant-system")
```

Comment:
Do not hard-code the name of the namespace, use `testconfig["service_protection"]["system_project"]` parameter